lxd/instance/drivers/driver_qemu.go

```diff
@@ -6407,46 +6409,85 @@ func (d *qemu) checkFeature(qemu string, args ...string) (bool, error) {
 		"-daemonize",
 		"-bios", filepath.Join(d.ovmfPath(), "OVMF_CODE.fd"),
 		"-pidfile", pidFile.Name(),
+		"-chardev", fmt.Sprintf("socket,id=monitor,path=%s,server=on,wait=off", monitorPath.Name()),
+		"-mon", "chardev=monitor,mode=control",
 	}
 
-	qemuArgs = append(qemuArgs, args...)
-
 	checkFeature := exec.Cmd{
 		Path: qemu,
 		Args: qemuArgs,
 	}
 
 	err = checkFeature.Start()
 	if err != nil {
-		return false, err // QEMU not operational. VM support missing.
+		return nil, err // QEMU not operational. VM support missing.
 	}
 
 	err = checkFeature.Wait()
 	if err != nil {
-		return false, nil // VM support available, but io_ring feature not.
+		return nil, err
+	}
+
+	monitor, err := qmp.Connect(monitorPath.Name(), qemuSerialChardevName, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	defer monitor.Disconnect()
+
+	var features []string
+
+	blockDevPath, err := os.CreateTemp("", "")
+	if err != nil {
+		return nil, err
+	}
+
+	defer func() { _ = os.Remove(blockDevPath.Name()) }()
+
+	// Check io_uring feature.
+	blockDev := map[string]any{
+		"node-name": d.blockNodeName("feature-check"),
+		"driver":    "file",
+		"filename":  blockDevPath.Name(),
+		"aio":       "io_uring",
+	}
+
+	err = monitor.AddBlockDevice(blockDev, nil)
+	if err != nil {
+		logger.Debug("Failed adding block device during feature check", logger.Ctx{"err": err})
```

Comment:
Can you change these to "... during VM feature check" otherwise these log messages,when shown in the log, will not be clearly related to VMs.